baselines/driving_smarts/v0/interaction_aware_motion_prediction.yaml

```diff
@@ -0,0 +1,7 @@
+agent:
+  interface:
+    action_space: "TargetPose" # we can only allow `RelativeTargetPose` and `TargetPose`
+    img_meters: 50 # Observation image area size in meters.
+    img_pixels: 112 # Observation image size in pixels.
+    observation_options: "unformatted"
```

Comment:
This does not seem to be the right place for the `observation_options` configuration since this is really an environment configuration rather than a per-agent configuration.

However, we do not expose the environment configuration necessary to use this configuration and it is related to interface (although not the agent interface.) I am somewhat conflicted.

So we have 2 options here:

a. Do per-agent configuration of observation formatting.
b. Expose environment level formatting.

Reply:
The discussion lead us to scrapping this configuration in place of using the `agent locator` of the agent directly. This should be possible because all of the agent interface configurations should already be contained in the agent spec that is returned.

The formatting will just be set to `"unformatted"` in the environment configuration within the benchmark instead. We will not do per-agent configuration of environment level action space formatting and environment level observation space formatting because that will make the environment spaces overly complex. Instead the agents should use wrappers to make the conversion.

This means that the utilities in `smarts/env/utils/[(action)|(observation)]_conversion.py` may need to be moved and the per-agent space conversion added into wrappers.

Reply:
Addressed in PR #1846 .